src/Workleap.Extensions.MediatR.Analyzers.Tests/Workleap.Extensions.MediatR.Analyzers.Tests.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Analyzer.Testing.XUnit" Version="1.1.1" PrivateAssets="all" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.4.0" PrivateAssets="all" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
-    <PackageReference Include="xunit" Version="2.6.1" />
+    <PackageReference Include="xunit" Version="2.4.2" />
```

Comment:
I was running into test failures so @asimmon directed me to downgrade XUnit for the analyzer tests to a more stable version which fixed the test failures.